### ISSUE TITLE
Web cypress test fail

### ISSUE THREAD

**Issue by Thykof** [massalabs/station-massa-wallet#834]
**Describe the bug**

https://github.com/massalabs/station-massa-wallet/actions/runs/6799616643/job/18486301874?pr=832
https://github.com/massalabs/station-massa-wallet/actions/runs/6799768770/job/18486784481?pr=832

**To Reproduce**

cd web-frontend
npm run cy:e2e

**Expected behavior**

all tests pass
**Additional context**
we commented out the send test to unblock merge.

**Comment by Thykof** [comment]
This has been solved in another PR
